### ISSUE TITLE
[Idea] multiple accounts, more scanning, faster scan refresh. Is it possible?

### ISSUE THREAD

**Issue by xan326** [AHAAAAAAA/PokemonGo-Map#1718]
I'm not a programmer, so I need help with this idea, if it's possible. 
I'm wanting to use multiple secondary accounts to enable more scanning in multiple places.
I'm running the app through Heroku, I set the step distance to 20, and the first ones turned red, then eventually disappeared. I'm assuming they timed out. I know in the video the dev says that it scans in a spiral pattern.
Well, for my idea, I want to place multiple locations in an isometric grid, as the scanned locations show up as a hexagon. I say -st 5 would be a good number to start with. But I'm wanting to place each location on the center of what would be the last location of -st 10. I could theoretically get the amount of scanning of -st 15 done within the time of -st 5 with 7 (center plus 6 outer) locations. To perfect this, it'd take some work knowing where exact centers are placed. I think I did my math correctly (long day and ill) but the individual scan areas will either border each other or overlap by one. 
Does anyone think this is possible? Also I'm wanting to do this within one map, like having separate account credentials within the script. 

TLDR; faster scanning, faster scan refresh, with multiple accounts. How do?

**Comment by KevinCollmer** [comment]
why didnt you just use Threading?

python ...runserver.py ... -t 25

**Comment by medghaim** [comment]
Please take support questions to https://discordapp.com/invite/Kzp4yky
